### ISSUE TITLE
Ingress rule for public control plane access is not created

### ISSUE THREAD

**Issue by santacriptus** [oracle-terraform-modules/terraform-oci-oke#433]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version and Provider Version

Terraform v1.0.11
+ provider registry.terraform.io/hashicorp/oci v4.53.0
### Terraform Configuration Files

control_plane_type         = "public"
control_plane_allowed_cidrs  = ["0.0.0.0/0"]

### Expected Behavior

can manage cluster with kubectl having ingress rule that allows access to control-plane subnet

### Actual Behavior
cannot manage cluster with kubectl, don't have ingress rule that allows access to control-plane subnet from my cidr
changing the value of control_plane_allowed_cidrs and executing terraform plan: terraform doesn't recognise the change 

### Steps to Reproduce

1. `terraform apply`
2. try to access your cluster with kubectl

**Comment by hyder** [comment]
Hi,

Thank you for logging this issue. Can I please check if you recently cloned the repo? If so, was it after issue #430 was closed?

**Comment by santacriptus** [comment]
Hi,

I cloned the repo on Nov 28 at 09:24 (GMT+2)

Please excuse me for duplicating the issue.

**Comment by hyder** [comment]
No worries at all. Can you please let us know your current commit hash?

**Comment by santacriptus** [comment]
I pulled the last change and now I am here:
commit 2eec72ebaab9bbdd17f444a45edf683c62948aca (HEAD -> main, origin/main, origin/HEAD)
Author: Ali Mukadam <ali.mukadam@oracle.com>
Date:   Mon Nov 29 17:49:59 2021 +1100
    fix: added 1 additional rule to allow control plane to be accessed by specified list of cidr blocks (#431)

Unfortunately I cannot test it soon because I made a lot of manual changes to my environment and I have to import them to my templates before any further terraform apply. I think we can close this issue for now.

Thank you so much for fast response and nice cooperation!
